docs/qa/AUTH-V2-FIXES-SUMMARY.md

```
@@ -0,0 +1,339 @@
+
+# 6. Validar en staging
+# - https://staging.roastr.ai/register
+# - https://staging.roastr.ai/login
+```
+
+### Post-Deploy Validation
+
+**Inmediatamente después del deploy:**
+
+1. Abrir DevTools console
+2. Ir a https://staging.roastr.ai/register
```

markdownlint-cli2 (0.20.0)

[warning] Bare URL used

(MD034, no-bare-urls)